index.html

```diff
@@ -97,6 +98,7 @@
     table.simple td {
         border-right: 1px solid #000;
     }
+    .issue[id^='issue-'] > *:not([role='heading']) { /* display: none; */ }
```

Comment:
It strips off pasting in the text taken from the issue description, but allows to provide own text description. With ```display: none``` it only links in the issue, without any text. At least I'd prefer keeping it this way, or of course I could remove the commented part.